.github/workflows/flink-ci.yml

```diff
@@ -93,7 +93,7 @@ jobs:
     strategy:
       matrix:
         jvm: [8, 11]
-        flink: ['1.13', '1.14', '1.15']
+        flink: ['1.13', '1.14', '1.15', '1.16']
```

Comment:
can you create a PR to remove 1.13? I think we typically remove the old version first before merging a new version.

Reply:
a new PR: #6103